prog/vnet/update_load_balancer_node.rb

```diff
@@ -39,7 +39,7 @@ def before_run
 
   def generate_lb_based_nat_rules
     public_ipv4 = vm.ephemeral_net4.to_s
-    public_ipv6 = vm.ephemeral_net6.to_s
+    public_ipv6 = vm.ephemeral_net6.nth(2).to_s
```

Comment:
Neat, although ... I'm a little concerned that in this tiny, tiny bit of code we ascribe special meaning to another address in the IPv6 space. I mean, it's great that we can, but probably need to follow up with some kind of abstraction that centralizes and describes how the address space is used.

Reply:
`vm.ephemeral_net6.nth(2)` is the public ip that we assign to customer, so, we are not picking another address from the address space for load balancing. We can, but I did not do simply because of the reason you describe. I want to keep it as a future improvement.

Reply:
oh what was I thinking. I was thinking you were working with the special internal half of the space. nevermind.